activesupport/lib/active_support/duration.rb

```diff
@@ -86,7 +93,7 @@ def eql?(other)
       Duration === other && other.value.eql?(value)
     end
 
-    def hash
+    def hash # :nodoc:
```

Comment:
We don't have to say that. People know that already because we're overriding it. It's a standard OO convention to override a superclass method for specialization, so we don't have to tell people that.